test/ec2_test.exs

```diff
@@ -0,0 +1,94 @@
+defmodule AntikytheraAws.Ec2.ClusterConfigurationTest do
+  use Croma.TestCase
+  alias Antikythera.Httpc
+
+  @region_args                    ["--region", Application.fetch_env!(:antikythera_aws, :region)]
+  @auto_scaling_group_name        Application.fetch_env!(:antikythera_aws, :auto_scaling_group_name)
+  @availability_zone_metadata_url "http://169.254.169.254/latest/meta-data/placement/availability-zone"
+  @noda_a_instance_id             "i-0d87927817101e3d1"
```

Comment:
Thanks for pointing! and because this project is an OSS project, is it ok to discuss it in English?
Fixed in https://github.com/access-company/antikythera_aws/pull/8/commits/0d6f0c0652ebc504903a4a720d146f6a1073b23b

Reply:
Thanks for pointing out! I will fix to use English.